### DISCUSSION TITLE
show unavailable only

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#7846] [Configuration] [answered]
would be very nice if we could have the entities list on `config/entities` only show unavailable entities, when the box 'unavailable' was ticked. Right now, it shows all entities, including the unavailable entities, which makes it that much more difficult to find the unavailable ones. 
Maybe add a tick box for unavailable only?

<img width="347" alt="Schermafbeelding 2020-11-29 om 12 18 16" src="https://user-images.githubusercontent.com/33354141/100540365-524b6a00-323d-11eb-9b3b-9e3815696b82.png">

**Comment by bramkragten** [comment, marked as answer]
You know you can order by status? All unavailable ones will be together then

**Reply by bramkragten** [reply]
A more hidden feature, enter `unavailable` in the search input

**Reply by Mariusthvdb** [reply]
sorry, accidentally unmarked this, while I wanted to mark the second (hidden feature) as answer. Cant I (Author) mark as answer, or is this only for Maintainers

**Reply by iantrich** [reply]
Doesn't look like you can mark a reply in as an answer, only the initial comment.

**Reply by Mincka** [reply]
Maybe it can be more intuitive if there are inclusive choices like "Show enabled entities" in the menu?

From a UX perspective, I am also expecting to show **only** unavailable entities when using this. It makes it harder to tick the box, sort and scroll to find them. Maybe that's the wording for the tick boxes that's wrong. Maybe "**Include** disabled entities" would be clearer for this behavior.

Another possibility would be helper links to set the search field with `disabled`, `unavailable` or `read-only` (the latest does not work currently). It would highlight the "hidden feature".

**Comment by Mariusthvdb** [comment]
thanks Bram, this will do for now, ;-) searching for unavailable working nicely.
